commitlint.config.js

```diff
@@ -1 +1,25 @@
-module.exports = { extends: ['@commitlint/config-conventional'] };
+module.exports = {
+    extends: ['@commitlint/config-conventional'],
+
+    rules: {
+        'body-max-line-length': [1, 'always', 80],
+        'header-max-length': [1, 'always', 72],
+        'scope-case': [2, 'never', ['kebab-case', 'snake-case']],
+        'subject-case': [
+            2,
+            'never',
+            [
+                'upper-case',
+                'camel-case',
+                'kebab-case',
+                'pascal-case',
+                'snake-case',
+            ],
+        ],
+        'type-enum': [
+            2,
+            'always',
+            ['chore', 'docs', 'feat', 'fix', 'refactor', 'style', 'test'],
```

Comment:
Prettier formatted array above differently than this one. I don't know why...

Reply:
Probably line length.